Management file change + weekly report week 3

GuillaumeDespaux opened the pull request:
Some documents have been improved
Adding the weekly report for week 3

GuillaumeDespaux committed:
Risks & Assumtion + Weekly

GuillaumeDespaux committed:
File navigation improvement

GuillaumeDespaux committed:
Update weekly_report_3.md

GuillaumeDespaux committed:
Update management_artifacts.md

Adding WBS + Gantt Chart part
Rework KPI's architecture

GuillaumeDespaux committed:
Risks & Assumption errors file

moving the remaining content of risks & assumption in the right file

Enonome reviewed (changes requested):
In management_artifacts.md:
> - 1st sentence:  Maybe "Here is the WBS. You ..." would be more appropriate
> - Missing the link to your Actual Gantt Chart
> - Subsection "Design & Bonus", "Each task is separated [...] point. For example ..." is probably better
> - In each subsection from "Files" to "Game/Bonus", "there" -> "here" ; "witch" -> "which" ; "including" -> "included"
> - In the Calculation subsection, Idk why but, the $$ are displayed instead of formatted

In project_charter.md:
> Ok for me

In the weekly reports:
> Ok for me

GuillaumeDespaux committed:
Errors fixing for PR

Correcting erros due to:
https://github.com/algosup/2024-2025-project-1-fpga-team-2/pull/12#pullrequestreview-2367238927

GuillaumeDespaux committed:
Typo rectification

GuillaumeDespaux committed:
Typo correct

Enonome reviewed (approved):
Ok for me
